modules/javafx.graphics/src/test/java/test/javafx/scene/layout/AnchorPaneTest.java

```diff
@@ -718,4 +725,56 @@ public void testBottomAnchoredMinSizeOverridden() {
         assertEquals(300, child.getWidth(), 1e-100);
         assertEquals(400, child.getHeight(), 1e-100);
     }
+
+    /**
+     * Tests the snapping of the {@link AnchorPane} with different scales.
+     *
+     * @param scale the scale which is used as render scale on the {@link Stage}
+     * @see <a href="https://bugs.openjdk.org/browse/JDK-8295078">JDK-8295078</a>
+     */
+    @ValueSource(doubles = { 1.0, 1.25, 1.5, 1.75, 2.0 })
+    @ParameterizedTest
+    void testAnchorPaneSnappingWithDifferentScales(double scale) {
+        double padding = 9.6;
+
+        StackPane child = new StackPane();
+        AnchorPane anchorpane = new AnchorPane(child);
+        anchorpane.setStyle("-fx-padding: " + padding + "px;");
```

Comment:
There might be two cases: when snapToPixel=true (this is the default case), and when =false.  Should we test the case snapToPixel=false?

Reply:
We can, but that should do the same as before. As before the values were not snapped at all. And that worked, I guess 😄